docs/dev/setup.rst

```diff
@@ -365,12 +363,13 @@ above in Server Setup) and if you have configured
 ``application-artemis.yml`` correctly, then you should be able to login
 with your TUM Online account.
 
-In case you encounter any problems regarding JavaScript heap memory leaks when executing ``yarn start`` or any other scripts from ``package.json``, you can change the memory limit parameter used in the ``webpack-ts`` script in ``package.json`` to the following:
+In case you encounter any problems regarding JavaScript heap memory leaks when executing ``npm run serve`` or any other scripts from ``package.json``,
+you can add a memory limit parameter used in the script in ``package.json`` to the following:
 
 ::
 
    # This local change in `package.json` should not be committed.
-   "webpack-ts": "cross-env NODE_OPTIONS=--max_old_space_size=5120 TS_NODE_PROJECT=\"tsconfig.webpack.json\" webpack" # possible higher values are 6144, 7168, and 8192
+   --max_old_space_size=5120 # possible higher values are 6144, 7168, and 8192
```

Comment:
For me it wasn't clear where to add this change. I tried to add it in the serve script but it didn't work.
What works for me is to change the start script like that: 
```
"start": "node --max_old_space_size=5120 ./node_modules/@angular/cli/bin/ng serve --hmr ",
```

Maybe we can add this in the documentation as another possible option since it took some time until I found this solution and it may be needed from other people as well.

Reply:
I'll add this point to the general comments in a [confluence page](https://confluence.ase.in.tum.de/display/ArTEMiS/Code+Review), as I cannot decide now if maybe the current version is not valid at all, so we should rather replace it with yours, or if we add your solution. Let's discuss this later on.